source/js/src/video.js.es6

```diff
@@ -1,52 +1,100 @@
 //= require "../lib/require.2.3.6.min.js"
 
-// NOTE: We use require.js to load Vimeo, then initialize Vimeo.
-// NOTE: …/player.js is preloaded in the <head>!
-require(['https://player.vimeo.com/api/player.js'], function(Player) {
+// Videos
+;(function() {
```

Comment:
I had a brief look and I think this whole function looks great.  Can't say I 100% follow it without really digging in, but yeah.